### PR TITLE
Convert VectorMap and RangeSelector topics to md

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

File: concepts/05 Widgets/VectorMap/40 End-User Interaction/End-User Interaction.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This section contains guides on the following end-user interaction features:
+
+- Zooming
+- Panning
+- Elements selection
+- Click handling
+- Hovering
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/VectorMap/40 End-User Interaction/End-User Interaction.md]: Let's reorder the items as the articles and add links to the articles.

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/VectorMap/40 End-User Interaction/End-User Interaction.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+This section contains guides on the following end-user interaction features:
+
+- [Panning](/Documentation/Guide/Widgets/VectorMap/End-User_Interaction/#Panning)
+- [Zooming](/Documentation/Guide/Widgets/VectorMap/End-User_Interaction/#Zooming)
+- [Hovering](/Documentation/Guide/Widgets/VectorMap/End-User_Interaction/#Hovering)
+- [Click handling](/Documentation/Guide/Widgets/VectorMap/End-User_Interaction/#Click_Handling)
+- [Elements selection](/Documentation/Guide/Widgets/VectorMap/End-User_Interaction/#Selection)
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/VectorMap/40 End-User Interaction/End-User Interaction.md]:
```suggestion
- [Element selection](/Documentation/Guide/Widgets/VectorMap/End-User_Interaction/#Selection)
```

**Review by RomanTsukanov — APPROVED** [review]

File: concepts/05 Widgets/RangeSelector/10 Visual Elements/Visual Elements.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-The image below is the **RangeSelector** element map that can be helpful to you as you begin learning about dxRangeSelector capabilities. <b>Hover</b> over the image and you will learn how different **RangeSelector** elements are called. <b>Click an element</b> to navigate to a topic with more details.
+The image below displays the **RangeSelector** element map. **Hover** over the image and to see the **RangeSelector** element names. Click an element to navigate to a topic with more details.
```
Comment (dirk-pieterse) [inline concepts/05 Widgets/RangeSelector/10 Visual Elements/Visual Elements.md]:
```suggestion
The image below displays the **RangeSelector** element map. **Hover** over the image to show the **RangeSelector** element names. Click an element to navigate to a topic with more information.
```

File: concepts/05 Widgets/VectorMap/40 End-User Interaction/End-User Interaction.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+This section contains guides on the following end-user interaction features:
```
Comment (dirk-pieterse) [inline concepts/05 Widgets/VectorMap/40 End-User Interaction/End-User Interaction.md]:
```suggestion
This section contains guides on the following user interaction features:
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**